components/core/tools/scripts/lib_install/openssl.sh

```
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# Exit on error
+set -e
+
+cUsage="Usage: ${BASH_SOURCE[0]} <version> [prefix] [openssldir]"
+if [ "$#" -lt 1 ] ; then
+    echo $cUsage
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)